File: robin/python/programmers/level1/study9/week_one.py
def solution(price, money, count):
    return max(sum([price * i for i in range(1, count + 1)]) - money, 0)


if __name__ == '__main__':
    price = 3
    money = 20
    count = 4
    print(solution(price, money, count))

File: robin/java/src/programmers/level1/study9/WeekOne.java
package programmers.level1.study9;

import java.util.stream.LongStream;

public class WeekOne {

    public static long solution(int price, long money, int count) {
        return Math.max(LongStream.range(1, count + 1).map(x -> x * price).sum() - money, 0);
    }

    public static void main(String[] args) {
        int price = 3;
        int money = 20;
        int count = 4;

        System.out.println(solution(price, money, count));
    }
}
